gtk4/src/auto/icon_theme.rs

```diff
@@ -154,6 +166,13 @@ impl IconTheme {
         }
     }
 
+    #[doc(alias = "gtk_icon_theme_set_resource_path")]
+    pub fn set_resource_path(&self, path: &[&str]) {
```

Comment:
That's weird. It supposed to use `&[P]` with `P: AsRef<Path>`

Reply:
no, the default is "/org/gtk/libgtk/icons/", so that binding looks correct to me.

Reply:
Never mind me, I got confused by the `_path` part of the function name.